week-1/interpret/time-format.js

```diff
@@ -14,16 +14,22 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// Ans. 7
 
 // b) How many function calls are there?
+// Ans. None
 
 // c) Using documentation on MDN, explain what the expression movieLength % 60 represents
+//Ans. This expression is providing us with the remainder of any seconds which are less than 60.
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+//Ans. It is calculating the minutes of movie which are less than 60 after taking out any extra seconds which are less than 60
 
 // e) What do you think the variable result represents? Can you think of a better name for this variable?
+// Ans. It represents the total length of movie broken down in hour, mins and seconds. We can also call it MovieRuntime
```

Comment:
Thinking about the names of the variables that form the expression, is there anything we can say to be more specific about what the variable represents? I think this variable name could be easily confused with the runtime of the whole movie

Reply:
HoursMinutesSeconds , possibly better than runtime?